src/iOS.Autofill/CredentialProviderViewController.cs

```diff
@@ -223,6 +223,7 @@ public override void PrepareForSegue(UIStoryboardSegue segue, NSObject sender)
                     else if (navController.TopViewController is LockPasswordViewController passwordViewController)
                     {
                         passwordViewController.CPViewController = this;
+                        passwordViewController.LaunchHomePage = () => DismissViewController(false, () => LaunchHomePage()); ;
```

Comment:
⛏️ Remove double ";"
```suggestion
                        passwordViewController.LaunchHomePage = () => DismissViewController(false, () => LaunchHomePage());
```